src/components/item/publish/ItemPublishConfiguration.js

```diff
@@ -128,10 +164,12 @@ const ItemPublishConfiguration = ({
   };
 
   const handleRefresh = () => {
-    queryClient.invalidateQueries(buildItemValidationAndReviewsKey(itemId));
+    queryClient.invalidateQueries(buildItemValidationAndReviewKey(itemId));
   };
 
   const publishItem = () => {
+    const publishedTag = getTagByName(tags, SETTINGS.ITEM_PUBLISHED.name);
+    const publicTag = getTagByName(tags, SETTINGS.ITEM_PUBLIC.name);
```

Comment:
So if we handle public in the other dropdown, no need to set the public tag again.